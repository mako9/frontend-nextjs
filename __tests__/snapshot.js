// __tests__/snapshot.js

import { render, cleanup } from '@testing-library/react'
import Home from '../pages/index'

afterEach(cleanup);

jest.mock('next-auth/react', () => {
  return {
    useSession: jest.fn(() => {
      return {
        user: {
          email: 'test@example.com',
          name: 'Test User',
        },
        signin: jest.fn(),
        signout: jest.fn(),
      };
    }),
  };
});

it('renders homepage unchanged', () => {
  const { container } = render(
      <Home allCommunitiesData={[]} />
  )
  expect(container).toMatchSnapshot()
})