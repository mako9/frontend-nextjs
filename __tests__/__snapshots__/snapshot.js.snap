// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div>
    <section
      class="headingMd"
    >
      <p>
        index.header
      </p>
    </section>
    <section
      class="headingMd padding1px"
    >
      <ul
        class="list"
      >
        <div>
          <h1>
            index.availableItems
          </h1>
          <div
            class="gallery"
          />
        </div>
      </ul>
    </section>
    <section
      class="headingMd padding1px"
    >
      <h2
        class="headingLg"
      >
        index.availableCommunities
      </h2>
      <ul
        class="list"
      />
    </section>
  </div>
</div>
`;
